README.md

```diff
@@ -4,12 +4,11 @@ This project was created to collect [Amazon ECS](https://aws.amazon.com/ecs) log
 
 The following functions are supported:
 
-* Supported OS: Amazon Linux, Amazon Linux 2, Red Hat Enterprise Linux 7, CentOS, Debian 8, Ubuntu 14.04, Ubuntu 16.04, Ubuntu 18.04
```

Comment:
I think there is still value in keeping a list of supported OS as everyone coming to use the tool might not be aware of *systemd init systems*. 

We can say something like _Operating System's using system init system are supported including Amazon Linux, Amazon Linux 2, Red Hat Enterprise Linux 7, CentOS, Debian 8, Ubuntu 14.04, Ubuntu 16.04, Ubuntu 18.04._ or something on similar lines.

Reply:
The script should work with any OS and Debug mode will only work with Systemd init systems as stated in the debug section. I would not like to give the impression it only works with a few OS.

Reply:
How about _should work with any OS that uses systemd init, some of the popular ones are ....._ 

I am insisting on keeping the list of OS as folks can instantly associate when they see _Ubuntu_ or _Linux_ but not the case when they see _systemd init_.

Reply:
What do you think if we mention that in the [enable-debug](https://github.com/fjromerom/ecs-logs-collector/blob/master/README.md#example-output-in-enable-debug-mode) section which is the only one affected:

"This mode only works on Amazon Linux OS and Systemd init systems such as RHEL 7 and Ubuntu 16.04."

Reply:
Yes that works 👍